src/main/java/org/antipathy/mvn_scalafmt/FormatMojo.java

```diff
@@ -43,12 +44,10 @@
       * then we consider the value in {@code <branch>} tag as a command to run and the output will be used as the actual branch */
     @Parameter(property = "format.branch", defaultValue = "master")
     private String branch;
-    @Parameter(readonly = true, defaultValue = "${project}")
+    @Parameter(property = "project", readonly = true, required = true, defaultValue = "${project}")
```

Comment:
compared to #149: added `property = "project"` as some online sources insist that it's required with the switch from `maven-project` to `maven-core`.

however, i also tested with setting only one of `property = "project"` and `defaultValue = "${project}"`, and all three versions seem to work. looks like this was not the culprit, but decided to keep both just in case.